### DISCUSSION TITLE
Newbie needs help

### DISCUSSION THREAD

**Discussion by JamesTanner651** [AutomaApp/automa#420] [Q&A]
Hey people. I am loving automa thank you @Kholid060 for creating this. 

I just want to know how, where, in extension's JS, I can add google sheet ranges instead of opening extension in the browsers. 

Thanks in advance

**Comment by Kholid060** [comment]
In Automa source code, open the `src/background/workflow-engine/blocks-handler/handler-google-sheets.js` file. And below the `result` variable add `data.range = 'sheetRange'`

![sublime_text_16LqUMKJst](https://user-images.githubusercontent.com/22908993/160983760-97193088-2eef-4c55-97e1-f4364962185d.png)

**Reply by JamesTanner651** [reply]
How can I add this code in Extension File rather than adding it to the source file? @Kholid060

**Reply by Kholid060** [reply]
I don't think you can do that because the code in the extension file is has been minified
